### PR TITLE
Update README to reactphp/http v1.0.0 and add improved HTTP examples

### PR DESCRIPTION
**Pull request by SimonFrings**

### CODE REVIEW

File: examples/01-http-request.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+// A simple example which uses an HTTP client to request https://example.com/ through an HTTP CONNECT proxy.
+// You can use any kind of proxy, for example https://github.com/leproxy/leproxy and excecute it like this:
+//
+// $ php leproxy-latest.php
+//
+// To run the example, go to the project root and run:
+//
+// $ php examples/01-http-requests.php
+//
+// Make sure you're referencing the right proxy adress inside the ProxyConnector
```
Comment (clue) [inline examples/01-http-request.php]:
```suggestion
// Make sure you're referencing the right proxy address inside the ProxyConnector
```

File: README.md [hunk 2]
```diff
@@ -203,12 +203,28 @@ $connector->connect('tls://smtp.googlemail.com:465')->then(function (ConnectionI
...
+    'dns' => false
+));
+
+$browser = new React\Http\Browser($loop, $connector);
+
+$browser->get('https://example.com/')->then(function (Psr\Http\Message\ResponseInterface $response) {
+    var_dump($response->getHeaders(), (string) $response->getBody());
+}); 
+```
+See also [ReactPHP's HTTP client](https://github.com/reactphp/http#client-usage) for more details.
```
Comment (clue) [inline README.md]:
```suggestion

See also [ReactPHP's HTTP client](https://github.com/reactphp/http#client-usage) for more details.
```

File: examples/11-proxy-raw-https-protocol.php [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
+//
+// To run the example, go to the project root and run:
+//
+// $ php examples/11-proxy-raw-https-protocol.php
+//
 // The proxy can be given as first argument and defaults to localhost:8080 otherwise.
 //
 // For illustration purposes only. If you want to send HTTP requests in a real
-// world project, take a look at https://github.com/clue/reactphp-buzz#http-proxy
+// world project, take a look at example-01
```
Comment (clue) [inline examples/11-proxy-raw-https-protocol.php]:
```suggestion
// world project, take a look at example #01 and https://github.com/reactphp/http#client-usage. 
```

?

File: examples/12-optional-proxy-raw-https-protocol.php [hunk 4]
```diff
@@ -2,13 +2,17 @@
...
+// $ php examples/12-optional-proxy-raw-https-protocol.php
+//
 // The Proxy can be given as first argument or does not use a proxy otherwise.
 // This example highlights how changing from direct connection to using a proxy
 // actually adds very little complexity and does not mess with your actual
 // network protocol otherwise.
 //
 // For illustration purposes only. If you want to send HTTP requests in a real
-// world project, take a look at https://github.com/clue/reactphp-buzz#http-proxy
+// world project, take a look at example-01
```
Comment (clue) [inline examples/12-optional-proxy-raw-https-protocol.php]:
```suggestion
// world project, take a look at example #01 and https://github.com/reactphp/http#client-usage. 
```

?

**Review by clue — CHANGES_REQUESTED** [review]
@SimonFrings Thanks for the update, add some minor remarks below :shipit:

File: examples/01-http-request.php (hunk 1 above)
Comment (SimonFrings) [reply examples/01-http-request.php]: Done :+1:

File: README.md (hunk 2 above)
Comment (SimonFrings) [reply README.md]: Done :+1:

File: examples/11-proxy-raw-https-protocol.php (hunk 3 above)
Comment (SimonFrings) [reply examples/11-proxy-raw-https-protocol.php]: Done :+1:

File: examples/12-optional-proxy-raw-https-protocol.php (hunk 4 above)
Comment (SimonFrings) [reply examples/12-optional-proxy-raw-https-protocol.php]: Done :+1:

**Review by Degra1991 — APPROVED** [review]

**Review by clue — APPROVED** [review]
@SimonFrings Thanks for the update, changes LGTM! :shipit:

**Pull request merged**